EIPS/eip-5192.md

```diff
@@ -34,6 +34,9 @@ A token with a `uint256 tokenId` may be inseparably-nound to a receiving account
 pragma solidity ^0.8.0;
 
 interface IERC5192 {
+  /// @notice Emitted when locking status is changed
```

Comment:
I think we should add an `@dev` note on what should happen upon minting. E.g. when minting "a regular NFT", what should `bool status` be?